sdks/python/apache_beam/transforms/core.py

```diff
@@ -1548,6 +1548,7 @@ def to_runner_api(self, context):
         output_time=self.timestamp_combiner,
         # TODO(robertwb): Support EMIT_IF_NONEMPTY
         closing_behavior=beam_runner_api_pb2.ClosingBehavior.EMIT_ALWAYS,
+        OnTimeBehavior=beam_runner_api_pb2.OnTimeBehavior.FIRE_ALWAYS,
```

Comment:
This is really weird that in the generated protos this one field is CamelCase and not snake_case.